src/tests/components/passwordReset.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { PasswordResetComponent } from "../../components/passwordReset";
+
+let props = {
+  password: "",
+  passwordConfirm: "",
+  handleOnChange: jest.fn(),
+  errors: {
+    password: "",
+    passwordConfirm: ""
+  },
+  handleOnSubmit: jest.fn()
+};
+let instance;
+describe("passwordComponent", () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(<PasswordResetComponent {...props} />);
+    instance = wrapper.instance();
+  });
+  it("should render properly", () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+  it("should ", () => {
```

Comment:
endeavour to use meaningful names for your tests

Reply:
done

Reply:
done

Reply:
Please elaborate on how this is done, @anyatibrian